
From: Adrian Bunk <bunk@fs.tum.de>

  LD      .tmp_vmlinux1
sound/built-in.o(.text+0xfb4ae): In function `snd_gus_synth_new_device':
: undefined reference to `snd_seq_iwffff_init'
make: *** [.tmp_vmlinux1] Error 1

Signed-off-by: Adrian Bunk <bunk@fs.tum.de>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/sound/core/seq/instr/Makefile |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN sound/core/seq/instr/Makefile~alsa-gus-compile-error sound/core/seq/instr/Makefile
--- 25/sound/core/seq/instr/Makefile~alsa-gus-compile-error	2004-06-27 22:38:03.651218016 -0700
+++ 25-akpm/sound/core/seq/instr/Makefile	2004-06-27 22:38:03.655217408 -0700
@@ -19,5 +19,5 @@ sequencer = $(if $(subst y,,$(CONFIG_SND
 # Toplevel Module Dependency
 obj-$(call sequencer,$(CONFIG_SND_OPL3_LIB)) += snd-ainstr-fm.o
 obj-$(call sequencer,$(CONFIG_SND_OPL4_LIB)) += snd-ainstr-fm.o
-obj-$(call sequencer,$(CONFIG_SND_GUS_SYNTH)) += snd-ainstr-gf1.o snd-ainstr-simple.o
+obj-$(call sequencer,$(CONFIG_SND_GUS_SYNTH)) += snd-ainstr-gf1.o snd-ainstr-simple.o snd-ainstr-iw.o
 obj-$(call sequencer,$(CONFIG_SND_TRIDENT)) += snd-ainstr-simple.o
_
